src/app/shared/user/user.component.html

```diff
@@ -1,10 +1,11 @@
 <div class="card" class="{{user.isDeleted ? 'deleted-card' : 'active-card'}}">
-    <div class="user-name">{{user.firstName + " "+ user.lastName}}</div>
-    <p align="right">
-        <button 
-        class="{{showDetails ? 'btn btn-primary' : 
-            (user.isDeleted ? 'btn btn-success' : 'btn btn-danger')}}"
-        (click)="modifyStatus(user.id)">{{showDetails ? 'Details' : (user.isDeleted ? 'Activate' : 'Deactivate')}}
+    <div class="user-name">{{user?.firstName+" "+user?.lastName}}</div>
+    <div class="internal-container">
+        <button *ngIf="showDetails" class="btn btn-warning" (click)="editUserDetails(user)">Edit</button>
+        <p *ngIf="!showDetails"></p>
```

Comment:
This empty `p` tag can be removed

Reply:
That empty p tag is there for the div to be spaced between, as the edit button is only there for the  manage nav and it is justifying the content itself, but in case of create nav and delete nav there is no edit button, so the button doesn't move to the right end of the div, that's why I have a p tag over their to adjust the spacing

Reply:
you can have a look into the css, to know why I did so

Reply:
Okay.